rest/route/permissions.go

```diff
@@ -0,0 +1,42 @@
+package route
+
+import (
+	"context"
+	"net/http"
+
+	"github.com/evergreen-ci/evergreen"
+	"github.com/evergreen-ci/evergreen/rest/model"
+	"github.com/evergreen-ci/gimlet"
+)
+
+type permissionsGetHandler struct{}
+
+func (p *permissionsGetHandler) Factory() gimlet.RouteHandler                     { return p }
+func (p *permissionsGetHandler) Parse(ctx context.Context, r *http.Request) error { return nil }
+
+func GetPermissions(permissions []string) []model.APIPermission {
```

Comment:
I wouldn't make this function public (and I might put it as a private method on your type, just to make things more clear,)

Reply:
great idea

Reply:
I think private makes sense, but having it on the type less so. It doesn't use any data on the type and it feels strange to put such a generic function on a route handler

Reply:
I don't think non-route handlers are going to use the function, and I think it's a code organization thing, not a methodish thing. you don't have to give the reciever a variable, just to make that more clear.

Reply:
FWIW, i agree with John that it should be private but does not need to be a method on the `permissionsGetHandler` class since it does not perform any actions on that type.

Reply:
there's no actions possible on the type (because it doesn't have any attributes,) so really just putting methods on them is a namespaceing thing, so while I think your impulse is 100% correct in the general case, I'd still opt to put these as methods on the type here, because route is a big package.